File: tests/fixtures/languages/sample.ex
# Minimal Elixir fixture for language parse-validation tests.

defmodule Demo.Greeter do
  @moduledoc "Tiny greeter for fixture purposes."

  alias String.Chars

  def greet(label) do
    "Hello, " <> label
  end

  defp internal(value), do: value
end

defmodule Demo.Util do
  def add(a, b), do: a + b
end

File: tests/fixtures/languages/sample.rb
# Minimal Ruby fixture for language parse-validation tests.

require "set"

module Demo
  class Greeter
    def initialize(label)
      @label = label
    end

    def greet
      "Hello, #{@label}"
    end
  end

  def self.add(a, b)
    a + b
  end
end
